cypress/features/regression/button.feature

```diff
@@ -0,0 +1,83 @@
+Feature: Button component
+  I want to check Button component properties
+
+  Background: Open Button component page default
```

Comment:
Probably we could change the Background description like that
```suggestion
  Background: Open Button component default page
```

Reply:
fix